src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -35,9 +35,22 @@ import { saveProfilePicture } from 'src/utils/image'
 import Logger from 'src/utils/Logger'
 import { useAsyncKomenciReadiness } from 'src/verify/hooks'
 import { Statsig } from 'statsig-react-native'
+import { ADJECTIVES, NOUNS } from './constants'
 
 type Props = StackScreenProps<StackParamList, Screens.NameAndPicture>
 
+//Exported for testing purposes only
```

Comment:
you can also name it with a leading _ to indicate that it's private. That way, you can't import it without acknowledging that you're doing something wrong (otherwise you might import the function without reading the code comment)